rel/pref.cpp

```diff
@@ -286,12 +301,18 @@ void set_input_disp_center_location(bool on) {
 };
 u8 get_input_disp_color() { return s_pref.input_disp_color; }
 void set_input_disp_color(u8 idx) { s_pref.input_disp_color = idx; }
+u8 get_ball_color() { return s_pref.ball_color; }
+void set_ball_color(u8 idx) { s_pref.ball_color = idx; }
+u8 get_ape_color() { return s_pref.ape_color; }
+void set_ape_color(u8 idx) { s_pref.ape_color = idx; }
 bool get_input_disp_notch_indicators() { return get_bool_pref(BoolPref::InputDispNotchIndicators); }
 void set_input_disp_notch_indicators(bool on) {
     set_bool_pref(BoolPref::InputDispNotchIndicators, on);
 }
 bool get_input_disp_raw_stick_inputs() { return get_bool_pref(BoolPref::InputDispRawStickInputs); }
 void set_input_disp_raw_stick_inputs(bool on) { set_bool_pref(BoolPref::InputDispRawStickInputs, on); }
+bool get_() { return get_bool_pref(BoolPref::Marathon); }
```

Comment:
Oops?

Reply:
This error is not in my current version? this might be in the second smaller commit I made which fixed a typo. This is also the only comment that seems to have a yellow "outdated" mark above it?